### PR TITLE
Homework exercise: calculate

### PR DESCRIPTION
**Pull request by michantoniuk**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: calculate
🏅 Granted 6 XP for @michantoniuk

**Comment by ziobron** [comment]
⏱ 3 XP extra for @michantoniuk for on-time delivery

File: homework/calculate/calculate.hpp [hunk 1]
```diff
@@ -3,5 +3,19 @@
...
+        return std::to_string(first - second);
+    } else if (command == "multiply") {
+        return std::to_string(first * second);
+    } else if (command == "divide") {
+        if (second == 0) {
+            return "Division by 0";
+        } else {
+        return std::to_string(first / second); // first should be *1.0 to include floating point results, but the test verifies only ints
+        }
+    } else {
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:18]: If you return variable here, you don't need to use `else if`

File: homework/calculate/calculate.hpp [hunk 2]
```diff
@@ -3,5 +3,19 @@
...
+        return std::to_string(first + second);
+    } else if (command == "subtract") {
+        return std::to_string(first - second);
+    } else if (command == "multiply") {
+        return std::to_string(first * second);
+    } else if (command == "divide") {
+        if (second == 0) {
+            return "Division by 0";
+        } else {
+        return std::to_string(first / second); // first should be *1.0 to include floating point results, but the test verifies only ints
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:16]: It's better to write comment above

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**